nss-day-cohort-22/company-website-jagged-ferrets
# Km products

# To test: 
1. Five products with images
1. Hover effect on cards
1. Button should add sku and qty to shoppingcart on local storage
1. shoppingcart should clear when refreshing local storage
